tests/ga/test_extension.py

```diff
@@ -567,6 +584,318 @@ def test_ext_handler_reporting_status_file(self, *args):
 
         self.assertEquals(expected_status_json, actual_status_json)
 
+    def test_handler_configuration_complete_configuration(self, *args):
+        data = '''{
+  "name": "ExampleHandlerLinux",
+  "version": 1.0,
+  "handlerConfiguration": {
+    "linux": {
+      "resources": {
+        "cpu": [
+          {
+            "cores": 2,
+            "limit_percentage": 25
+          },
+          {
+            "cores": 8,
+            "limit_percentage": 20
+          },
+          {
+            "cores": -1,
+            "limit_percentage": 15
+          }
+        ],
+        "memory": {
+          "max_limit_percentage": 20,
+          "max_limit_MBs": 1000,
+          "memory_pressure_warning": "low",
+          "memory_oom_kill": "enabled"
+        }
+      }
+    },
+    "windows": {}
+  }
+}
+'''
+        handlerConfig = HandlerConfiguration(json.loads(data))
+
+        self.assertEqual(handlerConfig.get_name(), "ExampleHandlerLinux")
+        self.assertEqual(handlerConfig.get_version(), 1.0)
+
+        cpu_limits = handlerConfig.get_cpu_limits_for_extension().cpu_limits
+        self.assertEquals(3, len(cpu_limits))  # 3 configurations sent
```

Comment:
Can you search and replace `assertEquals` with `assertEqual` everywhere? Primarily because of consistency and because `assertEquals` has been deprecated since Python 2.7.

Reply:
The `assertEquals` to `assertEqual` change is a big one (more than 100 matches). I think that could be a different PR as It'll cause a lot of changes.

Reply:
Sure, but please at least be consistent within your changes.

Reply:
I've only made changes in one file - test_extHandlers for the timebeing. I'll slowly make the changes in other places as well.